inc/cloudfront_media_purge/namespace.php

```diff
@@ -31,7 +31,8 @@ function purge_media_file_cache( int $media_id ) {
 	$upload_path      = wp_parse_url( $upload_url, PHP_URL_PATH );
 	$upload_path_info = pathinfo( $upload_path );
 	$items            = [];
-	$items[]          = $upload_path_info['dirname'] . '/' . $upload_path_info['filename'] . '*';
+	// Make sure the image filename is URL encoded, else CloudFront will not be able to purge it.
+	$items[]          = $upload_path_info['dirname'] . '/' . urlencode( $upload_path_info['filename'] ) . '*';
```

Comment:
Linter warning here; variables shouldn't be aligned any more, but even if they are, comments reset alignment.
```suggestion
	$items[] = $upload_path_info['dirname'] . '/' . urlencode( $upload_path_info['filename'] ) . '*';
```